src/js/index.js

```diff
@@ -0,0 +1,80 @@
+const espressoMenuForm = document.querySelector("#espresso-menu-form");
+const menuList = document.querySelector("#espresso-menu-list");
+
+function App() {
+  const addMenu = function(espressoNewMenu) {
+      const menuTemplate = `<li class="menu-list-item d-flex items-center py-2">
+      <span class="w-100 pl-2 menu-name">${espressoNewMenu}</span>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button"
+      >
+        수정
+      </button>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm menu-remove-button"
+      >
+        삭제
+      </button>
+    </li>`
+
+      menuList.insertAdjacentHTML("beforeend", menuTemplate);
+  }
+
+  const updateMenuCount = function() {
+    document.querySelector(".menu-count").innerText = `총 ${document.querySelectorAll("li.menu-list-item").length} 개`;
+  }
+
+  const registerMenu = function(e) {
+    e.preventDefault();
+    const espressoMenuAdd = espressoMenuForm.querySelector("#espresso-menu-name");
+    const espressoNewMenu = espressoMenuAdd.value;
+    if(espressoMenuAdd.value.trim() === ""){
+        alert("메뉴를 입력해주세요!");
+        return;
+    }
+    espressoMenuAdd.value = "";
+    addMenu(espressoNewMenu);
+    updateMenuCount();
+}
+
+const updateMenuName = function(e) {
+    const espressoMenuRename = e.target.closest("li").querySelector("span.menu-name");
+    const menuRenamed = prompt("메뉴를 수정 하세요", espressoMenuRename.innerText);
+    if(menuRenamed != null){
+        return (espressoMenuRename.innerText = menuRenamed);
```

Comment:
```suggestion
        espressoMenuRename.innerText = menuRenamed;
```

위 라인에서 return 없이 위와 같이만 구현되어도 될 것 같아요!

혹시 제가 미쳐 파악하지 못한 return 문을 추가하신 이유가 있다면 알려주시면 감사하겠습니다! :)